pdearena/models/registry.py

```diff
@@ -367,6 +370,15 @@
             "diffmode": False,
         },
     },
+    "GCAFluidNet2d-32": {
+        "class_path": "pdearena.modules.twod_gcaunet.GCAFluidNet2d",
```

Comment:
We are missing this module?

Reply:
sorry forgot to add to the commit. It is pushed now